### PR TITLE
[2019.2.1] Porting metaproxy changes in #50183 to 2019.2.1

### PR DESCRIPTION
**Pull request by garethgreenaway**
### What does this PR do?
Porting metaproxy changes in #50183 to 2019.2.1

### What issues does this PR fix or reference?
N/A

### Tests written?
**[NOTICE] Bug fixes or features added to Salt require tests.**
Please review the [test documentation](https://docs.saltstack.com/en/latest/topics/tutorials/writing_tests.html) for details on how to implement tests into Salt's test suite.
Yes.  Proxy tests exist.

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by xeacott — APPROVED** [review]

**Comment by dwoz** [comment]
re-run full all

**Review by dwoz — APPROVED** [review]

File: salt/minion.py [hunk 1]
```diff
@@ -3301,6 +3307,21 @@ def _create_minion_object(self, opts, timeout, safe,
                            jid_queue=jid_queue)
 
 
+def _metaproxy_call(opts, fn_name):
+    log.debug('=== here ====')
```
Comment (twangboy) [inline salt/minion.py]: Is this leftover from some debugging?

File: salt/proxy/dummy.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 '''
-This is a dummy proxy-minion designed for testing the proxy minion subsystem.
+This is the a dummy proxy-minion designed for testing the proxy minion subsystem.
```
Comment (twangboy) [inline salt/proxy/dummy.py]: Probably need to remove `the`

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by dwoz** [comment]
@twangboy In the interest of making progress towards the release I am going to merge this.

@garethgreenaway Please make another PR to resolve @twangboy's comments.

**Pull request merged**